realtime/src/config.rs

```diff
@@ -25,12 +25,17 @@ pub struct XSynthRealtimeConfig {
     /// Default: `[9]`
     pub drums_channels: Vec<u32>,
 
-    /// Whether or not to use a threadpool to render individual keys' voices.
+    /// Controls the use a threadpool to render individual keys' voices.
+    /// When a value is set, the specified number of threads will be used
+    /// for that operation, while `None` means no per-key concurrency.
+    /// If the value is set to `Some(0)` then the number of threads will
+    /// be determined automatically by `rayon`.
+    ///
     /// Regardless, each MIDI channel uses its own thread. This setting
     /// adds more fine-grained threading per key rather than per channel.
     ///
-    /// Default: `false`
-    pub use_threadpool: bool,
+    /// Default: `None`
+    pub threadpool: Option<usize>,
```

Comment:
Same here on the Some(0)

Reply:
Btw I'd rather treat Some(0) as an invalid value that causes errors, rather than a magic number, because logically thinking, what does "yes I want 0 threads" mean

Reply:
> what does "yes I want 0 threads" mean

True, will use the same approach as the above.